staging/src/github.com/labring/lvscare/care/care.go

```diff
@@ -27,70 +29,141 @@ import (
 	"github.com/labring/sealos/pkg/utils/logger"
 )
 
-//VsAndRsCare is
-func (care *LvsCare) VsAndRsCare() {
-	lvs := BuildLvscare()
-	//set inner lvs
-	care.lvs = lvs
-	if care.Clean {
+// VsAndRsCare is
+func (care *LvsCare) VsAndRsCare() (err error) {
+	if care.lvs == nil {
+		care.lvs = BuildLvscare()
+	}
+
+	defer func() {
+		if err != nil {
+			return
+		}
+		err = care.cleanup()
+	}()
+
+	cleanVirtualServer := func() error {
 		logger.Info("lvscare deleteVirtualServer")
-		err := lvs.DeleteVirtualServer(care.VirtualServer, false)
+		err := care.lvs.DeleteVirtualServer(care.VirtualServer, false)
 		if err != nil {
-			logger.Info("virtualServer is not exist skip...: %v", err)
+			logger.Warn("virtualServer is not exist skip...: %v", err)
 		}
+		return err
+	}
+
+	if care.Clean {
```

Comment:
This parameter may not be correct. Previously, it was designed to automatically clear ipvs and route before execution. Is it of great significance to see this at present?

Reply:
I think if we specified this parameter, lvscare should clean ipvs rules before exiting?

Reply:
Yes, this parameter should indeed be like this. However, the previous code did not do this, but deleted the corresponding IPVS and routing before execution. If you want to do this, the previous functions also need to be supported.

Reply:
Seems `--clean` flag only useful when the last time lvscare run with flag `--run-once`. So in my understanding, this parameter is not so important as lvscare will takes care of the maintenance ipvs/route rules automatically (before exiting). 
BTW, the `previous functions` you mentioned, is this one https://github.com/labring/sealos/blob/main/cmd/sealctl/cmd/ipvs.go#L32 ?

Reply:
By default, IPVS and routing will be cleaned up for each execution. So this function can be used as the default. --clean's parameters can be kept now.

Reply:
left it as it is.